powerspikegg/frontend/src/app/home/searchbar.component.ts

```diff
@@ -1,15 +1,34 @@
 import { Component, OnInit } from '@angular/core';
+import { QuerySenderService } from '../services/query-sender.service';
+import {Router} from "@angular/router";
```

Comment:
Quoting issue; Also please stick to your import style ({<space>FooBar<space>} or {FooBar}). Here and on all other imports.